doc/changes/changelog.md

```diff
@@ -0,0 +1,3 @@
+# Changes
+
+* [1.0.0](changes_1.0.0.md)
```

Comment:
Please consider marking generated files as "generated" in file `.gitattributes`. Compare for example [release-droid](https://github.com/exasol/release-droid/blob/main/.gitattributes) or [extension-manager](https://github.com/exasol/extension-manager/blob/main/.gitattributes).